### PR TITLE
setting permissions for new files

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR solves the problem of write permission restrictions to the glsai user to files generated by the glsftp user. 

### Review:
- [x] Code approved by @Mropat 
- [x] Tests executed by @mayabrandi 
- [x] "Merge and deploy" approved by @mayabrandi 

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by mayabrandi** [comment]
We can wait a little with this one. Still debugging reason for permission restriction.

File: cg_lims/get/files.py [hunk 1]
```diff
@@ -8,11 +8,10 @@
 from typing import List
 
 
-def get_lims_log_file(lims: Lims, file_id: str) -> pathlib.Path:
+def get_log_content(lims: Lims, file_id: str) -> str:
     """Searching for a log Artifact with file_id. 
     
-    If the artifact is found, returning the path to the attached file. 
-    Otherwise returning the file_id."""
+    If the artifact is found, returning the the file content."""
```
Comment (Mropat) [inline cg_lims/get/files.py:14]: Perhaps comment on this function could be updated to better fit what it is doing right now?

**Review by Mropat — APPROVED** [review]
:+1:

**Comment by mayabrandi** [comment]
Testing from within step in the web interface and from command line:

Skärmavbild 2020-10-22 kl  11 09 20
 Skärmavbild 2020-10-22 kl  11 08 35

Skärmavbild 2020-10-22 kl  11 08 42


The log is being "appended to" without file permission errors. 👍

**Pull request merged**